src/ranch_acceptors_sup.erl

```diff
@@ -26,7 +26,8 @@ start_link(Ref, Transport, Logger) ->
 
 -spec init([term()]) -> {ok, {supervisor:sup_flags(), [supervisor:child_spec()]}}.
 init([Ref, Transport, Logger]) ->
-	TransOpts = ranch_server:get_transport_options(Ref),
+	TransOptsTemp = ranch_server:get_transport_options(Ref),
+	TransOpts = strip_usupported_options(TransOptsTemp),
```

Comment:
isn't it better to use |> notation?
TransOpts = Ref |>  ranch_server:get_transport_options() |> strip_usupported_options() /

Reply:
I'm not aware of piping in Erlang.